### ISSUE TITLE
ReCaptcha: Allow for custom URL

### ISSUE THREAD

**Issue by harsha16208** [primefaces/primereact#3921]
### Describe the feature you would like to see added

A feature to use a custom URL for the ReCaptcha component can help using ReCaptcha component where Google services are restricted

### Is your feature request related to a problem?

Our application when being used by countries like China is unable to use the ReCaptcha component as the ReCaptcha component uses Google Recaptcha and the country restricts usage of Google services

### Describe the solution you'd like

1. Suggestion to override your method by extending your Component

const addRecaptchaScript = () => {
            recaptchaScript.current = null;

            if (!window.grecaptcha) {
                let head = document.head || document.getElementsByTagName('head')[0];
                let script = document.createElement('script');

                script.src = 'https://www.google.com/recaptcha/api.js?render=explicit';
                script.async = true;
                script.defer = true;

                script.onload = () => {
                    if (!window.grecaptcha) {
                        // eslint-disable-next-line no-console
                        console.warn('Recaptcha is not loaded');

                        return;
                    }

                    window.grecaptcha.ready(() => {
                        init();
                    });
                };

                recaptchaScript.current = script;

                head.appendChild(recaptchaScript.current);
            }
        };

2. Passing Custom ReCaptcha URL as a prop to the component

**Comment by melloware** [comment]
So just to be clear you need to change this value correct?

```js
script.src = 'https://www.google.com/recaptcha/api.js?render=explicit';
```

**Comment by melloware** [comment]
Please check out my PR.

**Comment by harsha16208** [comment]
yes can you please help us to change the value of script.src

**Comment by melloware** [comment]
Fixed for 8.7.4

**Comment by harsha16208** [comment]
Thanks for adding this feature to your next release can I please know when
V8.7.4 is going to be released and meanwhile can you please suggest us any
temporary solution if possible
